spyder/utils/environ.py

```diff
@@ -47,41 +88,23 @@ def clean_env(env_vars):
     exception is raised, an empty string will be used.
     """
     new_env_vars = env_vars.copy()
-    for key, var in iteritems(env_vars):
-        if PY2:
-            # Try to convert vars first to utf-8.
-            try:
-                unicode_var = to_text_string(var)
-            except UnicodeDecodeError:
-                # If that fails, try to use the file system
-                # encoding because one of our vars is our
-                # PYTHONPATH, and that contains file system
-                # directories
-                try:
-                    unicode_var = to_unicode_from_fs(var)
-                except Exception:
-                    # If that also fails, make the var empty
-                    # to be able to start Spyder.
-                    # See https://stackoverflow.com/q/44506900/438386
-                    # for details.
-                    unicode_var = ''
-            new_env_vars[key] = to_binary_string(unicode_var, encoding='utf-8')
-        else:
-            new_env_vars[key] = to_text_string(var)
+    for key, var in env_vars.items():
+        new_env_vars[key] = str(var)
 
     return new_env_vars
 
 
 class RemoteEnvDialog(CollectionsEditor):
     """Remote process environment variables dialog."""
 
-    def __init__(self, environ, parent=None):
+    def __init__(self, environ, parent=None,
+                 title="Environment variables", readonly=True):
         super(RemoteEnvDialog, self).__init__(parent)
         try:
             self.setup(
                 envdict2listdict(environ),
-                title=_("Environment variables"),
-                readonly=True,
+                title=_(title),
```

Comment:
This doesn't work to translate strings, i.e. you can't use variables inside `_()` or strings for translations. You need to use a real strings here instead.